quickref/qsharp-quick-reference.tex

```diff
@@ -586,7 +586,7 @@
 
   \begin{keysref}{Control Flow}
     Iterate over \newline a range of numbers
-                       & \texttt{for (\emph{index} in \emph{range}) \{ }\newline 
+                       & \texttt{for \emph{index} in \emph{range} \{ }\newline 
                          \texttt{\hphantom{....}// Use integer \emph{index}} \newline
                          \texttt{\hphantom{....}...} \newline
                          \texttt{\}} \newline
```

Comment:
Please update the syntax in `e.g.` sections as well, since both for loops have syntax examples for them